### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by LiYueqian-James**
- Modified /_layout/default.html so that it can be reused in other pages
- Separated navigation bar and the home page header

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-RCSSA Website
-====================
+# RCSSA Website
```
Comment (LiYueqian-James) [inline README.md:1]: Just some formatting here...

File: _includes/home_header.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!-- Header -->
```
Comment (LiYueqian-James) [inline _includes/home_header.html:1]: header is split into home_header.html and nav_bar.html(see below)

File: _layouts/default.html [hunk 3]
```diff
@@ -1,16 +1,13 @@
 <!DOCTYPE html>
```
Comment (LiYueqian-James) [inline _layouts/default.html:1]: To use the default layout, other page just needs to specify the front matter:
---
layout: default
sections:
- name: section1.html
- name: section2.html
---

File: index.html [hunk 4]
```diff
@@ -1,5 +1,10 @@
 ---
 layout: default
```
Comment (LiYueqian-James) [inline index.html:2]: the front matters to use /_layouts/default.html

**Review by Tr0py — APPROVED** [review]
This is a great optimization! Made the structure much clearer.

**Pull request merged**